cartridges/int_adyen_SFRA/cartridge/templates/default/checkout/billing/adyenPosForm.isml

```diff
@@ -2,6 +2,7 @@
 <link rel="stylesheet" type="text/css" href="${pdict.AdyenHelper.getCheckoutCSS()}"/>
 <iscontent type="text/html" encoding="off"/>
 <h3>${Resource.msg('terminal.selectTerminal', 'terminal', null)}</h3>
+<input id="terminalId" type="hidden" ${adyenPaymentFields.terminalId.attributes}/>
```

Comment:
Do we need a hidden field to submit fields in the form can't we do this by javascript ?